bundles/org.openhab.binding.unifiaccess/src/main/resources/OH-INF/addon/addon.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<addon:addon id="unifiaccess" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:addon="https://openhab.org/schemas/addon/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/addon/v1.0.0 https://openhab.org/schemas/addon-1.0.0.xsd">
+
+	<type>binding</type>
+	<name>UnifiAccess Binding</name>
+	<description>This is the binding for UnifiAccess.</description>
+
```

Comment:
Can you add the `connection` tag ?

Does the controller announce itself on mDNS? Or any other mechanism that can be used for the binding suggestion finder ?